features/jmx-config-generator/src/main/java/org/opennms/features/jmxconfiggenerator/jmxconfig/JmxDatacollectionConfiggenerator.java

```diff
@@ -390,7 +277,7 @@ private String createAndRegisterUniqueAlias(String originalAlias) {
         }
         //find alias crashes caused by cuting down alias length to 19 chars
         if (aliasList.contains(nameCutter.trimByCamelCase(uniqueAlias, 19))) {
-            logger.error("ALIAS CRASH AT :" + uniqueAlias + "\t as: " + nameCutter.trimByCamelCase(uniqueAlias, 19));
+            logger.warn("ALIAS CRASH AT :" + uniqueAlias + "\t as: " + nameCutter.trimByCamelCase(uniqueAlias, 19));
```

Comment:
Please change the "ALIAS CRASH AT" message. Use something 
like "Ambiguous attribute name "+uniqueAlias+"..."